lab_1/main.py

```diff
@@ -74,7 +135,19 @@ def detect_language(unknown_profile: dict, profile_1: dict, profile_2: dict, top
     :param top_n: a number of the most common words
     :return: a language
     """
-    pass
+    if not isinstance(unknown_profile, dict) or not isinstance(profile_1, dict) or not isinstance(
+            profile_2, dict) or not isinstance(top_n, int):
+        return None
+    compare_1 = compare_profiles(unknown_profile, profile_1, top_n)
+    compare_2 = compare_profiles(unknown_profile, profile_2, top_n)
+    language = ""
+    if compare_1 > compare_2:
+        language += profile_1['name']
+    elif compare_2 > compare_1:
+        language += profile_2['name']
+    else:
+        language += sorted([profile_1['name'], profile_2['name']])[0]
```

Comment:
explain what happens here in writing

Reply:
compare compare_1 and compare_2, if compare_1 > compare_2 or compare_1 < compare_2  func returns the appropriate profile name by key, if compare_1 = compare_2 - create a list, sort the names of profiles and take the first element in alphabetical order

Reply:
I don't understand why you need to add a new name to an empty string